internal/api/v1/stage.go

```diff
@@ -142,11 +142,11 @@ func (hc ApplicationsController) Stage(w http.ResponseWriter, r *http.Request) A
 
 	log.Info("staging app", "org", org, "app", req)
 
-	cs, err := tekton.NewForConfig(cluster.RestConfig)
+	tc, err := cluster.ClientTekton()
```

Comment:
I implemented a function for the "is it staging?" check here: https://github.com/epinio/epinio/pull/860/files#diff-44a311773090a2abd70d10d9f026c5d98e1a8f88f37381e50f8b4a30fe784854R100

I can fix on rebase though, no worries.